pkg/operator/controllers/routefix/routefix_controller.go

```diff
@@ -24,25 +26,32 @@ import (
 	aroclient "github.com/Azure/ARO-RP/pkg/operator/clientset/versioned"
 	"github.com/Azure/ARO-RP/pkg/operator/controllers"
 	"github.com/Azure/ARO-RP/pkg/util/dynamichelper"
+	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
 //RouteFixReconciler is the controller struct
 type RouteFixReconciler struct {
 	kubernetescli kubernetes.Interface
 	securitycli   securityclient.Interface
+	configcli     configclient.Interface
 	arocli        aroclient.Interface
 	restConfig    *rest.Config
 	log           *logrus.Entry
+	verFixed      *version.Version
 }
 
 //NewReconciler creates a new Reconciler
-func NewReconciler(log *logrus.Entry, kubernetescli kubernetes.Interface, securitycli securityclient.Interface, arocli aroclient.Interface, restConfig *rest.Config) *RouteFixReconciler {
+func NewReconciler(log *logrus.Entry, kubernetescli kubernetes.Interface, securitycli securityclient.Interface, configcli configclient.Interface, arocli aroclient.Interface, restConfig *rest.Config) *RouteFixReconciler {
+	verFixed, _ := version.ParseVersion("4.7.15")
```

Comment:
Should we bump to this version in `pkg/util/version/const.go` as well? it currently refers to 4.7.12. I think it is a good idea to start creating clusters from a version which contains this fix.

Reply:
Separate PR inprogress.